addons/mail/static/src/discuss/core/web/discuss_sidebar_categories.xml

```diff
@@ -0,0 +1,73 @@
+                <span class="btn-sm p-0 text-uppercase fw-bolder"><t t-esc="category.name"/></span>
+            </div>
+            <div class="flex-grow-1"/>
+            <div class="d-flex me-3">
+                <i t-if="category.canView" t-attf-class="fa fa-cog {{ hover_class }}" title="View or join channels" t-on-click="() => this.openCategory(category)" role="img"/>
+                <i t-if="category.canAdd and category.isOpen" class="o-mail-DiscussSidebarCategory-add" t-attf-class="fa fa-plus {{ hover_class }} ms-1" t-on-click="() => this.addToCategory(category)" t-att-title="category.addTitle" role="img"/>
+            </div>
+            <div t-if="!category.isOpen and threadService.getDiscussSidebarCategoryCounter(category.id) > 0" class="o-mail-DiscussSidebar-badge badge rounded-pill me-3 o-discuss-badge fw-bold">
+                <t t-esc="threadService.getDiscussSidebarCategoryCounter(category.id)"/>
+            </div>
+        </div>
+        <t t-if="category.isOpen">
+            <div t-if="state.editing === category.id" class="p-2" t-ref="selector">
+                <ChannelSelector category="category" onValidate.bind="stopEditing" autofocus="true"/>
+            </div>
+            <t t-foreach="filteredThreads(category)" t-as="threadLocalId" t-key="threadLocalId" t-call="mail.DiscussSidebarChannel">
+                <t t-set="thread" t-value="store.threads[threadLocalId]"/>
+            </t>
+        </t>
+        <t t-elif="category.threads.includes(store.discuss.threadLocalId)" t-call="mail.DiscussSidebarChannel">
+            <t t-set="threadLocalId" t-value="store.discuss.threadLocalId"/>
+            <t t-set="thread" t-value="store.threads[threadLocalId]"/>
+        </t>
+    </t>
+
+    <t t-name="mail.DiscussSidebarChannel" owl="1">
+        <t t-set="counter" t-value="threadService.getCounter(thread)"/>
+        <button class="o-mail-DiscussSidebarChannel o-mail-DiscussSidebar-item btn btn-secondary d-flex align-items-center w-100 px-0 py-2 border-0 rounded-0 text-reset"
+            t-att-class="{
+                'o-bg-inherit': threadLocalId !== store.discuss.threadLocalId,
+                'o-active': threadLocalId === store.discuss.threadLocalId,
+                'o-unread': thread.message_unread_counter > 0,
+            }"
+            t-on-click="(ev) => this.openThread(ev, thread)"
+        >
+            <div class="o-bg-inherit position-relative d-flex ms-4 flex-shrink-0" style="width:30px;height:30px">
+                <img class="w-100 h-100 rounded" t-att-src="thread.imgUrl" alt="Thread Image"/>
+                <ThreadIcon t-if="thread.type === 'chat' or (thread.type === 'channel' and !thread.authorizedGroupFullName)" thread="thread" size="'small'" className="'o-mail-DiscussSidebarChannel-threadIcon position-absolute bottom-0 end-0 p-1 me-n1 mb-n1 d-flex align-items-center justify-content-center rounded-circle o-bg-inherit'"/>
+            </div>
```

Comment:
Could be nice in future PR to define a `SidebarItem` component with slots, so that we have less duplicated code and we can easily reuse style for all mailboxes and channels.
I forgot sometimes to adapt style in both `Mailbox` and `CategoryItem` when it was in the same file. Now that it's in 2 different files, this might happen more frequently.